### PR TITLE
Fix get nodes info method

### PR DESCRIPTION
**Pull request by ahmedelsayed-93**
https://github.com/gig-projects/org_quality/issues/645

### CODE REVIEW

File: functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from api_testing.grid_apis.apis.nodes_apis import NodesAPI
+from unittest import TestCase
+
+def get_node_physical_ip(nodeid):
+    response = nodes_api.get_nodes_nodeid_nics(nodeid)
+    tc.assertEqual(response.status_code, 200)
```
Comment (islamTaha12) [inline functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py]: why should we assert here? I though it isn't a test case

File: functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        nic_hwaddr = nic['hardwareaddr'].replace(':', '') 
+        if nic_hwaddr == nodeid:
+            ip = nic['addrs'][0]
+            return ip[:ip.rfind('/')]
+
+def get_node_info():
+    nodes_api = NodesAPI()
+    nodes_info = []
+    response = nodes_api.get_nodes()
+    tc.assertEqual(response.status_code, 200)
```
Comment (islamTaha12) [inline functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py]: same comment

File: functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py (hunk 1 above)
Comment (ahmedelsayed-93) [inline functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py]: done

File: functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py (hunk 2 above)
Comment (ahmedelsayed-93) [inline functional_testing/Grid_API_Testing/api_testing/testcases/__init__.py]: done

File: functional_testing/Grid_API_Testing/api_testing/testcases/testcases_base.py [hunk 3]
```diff
@@ -14,7 +15,7 @@ def __init__(self, *args, **kwargs):
         self.utiles = Utiles()
         self.nodes_api = NodesAPI()
         self.config = self.utiles.get_config_values()
```
Comment (islamTaha12) [inline functional_testing/Grid_API_Testing/api_testing/testcases/testcases_base.py]: update this line by master one

**Pull request merged**